deploy.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 
 if [ "${TRAVIS_BRANCH}" == "csi-v0.3" ] && [ "${TRAVIS_PULL_REQUEST}" == "false" ]; then
+    export RBD_IMAGE_VERSION='v0.3.1-canary'
```

Comment:
I think need to make changes in [makefile](https://github.com/ceph/ceph-csi/blob/csi-v0.3/Makefile#L18) also.

exporting the image version is not working for me locally

```
#!/bin/bash
    export RBD_IMAGE_VERSION='v0.3.1-canary'
    export CEPHFS_IMAGE_VERSION='v0.3.1-canary'
    #docker login -u "${QUAY_IO_USERNAME}" -p "${QUAY_IO_PASSWORD}" quay.io
    make push-image-rbdplugin push-image-cephfsplugin
```
```
[🎩︎]mrajanna@localhost ceph-csi $]./deploy.sh 
rbd    image settings: quay.io/cephcsi/rbdplugin version v0.3.0
cephfs image settings: quay.io/cephcsi/cephfsplugin version v0.3.0
if [ ! -d ./vendor ]; then dep ensure -vendor-only; fi
CGO_ENABLED=0 GOOS=linux go build -a -ldflags '-extldflags "-static"' -o  _output/rbdplugin ./rbd
cp _output/rbdplugin  deploy/rbd/docker
docker build -t quay.io/cephcsi/rbdplugin:v0.3.0 deploy/rbd/docker
Sending build context to Docker daemon  37.94MB
Step 1/8 : FROM centos:7
 ---> 1e1148e4cc2c
Step 2/8 : LABEL maintainers="Kubernetes Authors"
 ---> Using cache
 ---> aad70d68c2f8
Step 3/8 : LABEL description="RBD CSI Plugin"
 ---> Using cache
 ---> 5badf886fe58
Step 4/8 : ENV CEPH_VERSION "mimic"
 ---> Using cache
 ---> b080a5a6371a
Step 5/8 : RUN yum  install -y centos-release-ceph &&     yum install -y ceph-common e2fsprogs xfsprogs rbd-nbd &&     yum clean all
 ---> Using cache
 ---> e0e4552a4cc3
Step 6/8 : COPY rbdplugin /rbdplugin
 ---> Using cache
 ---> 3997625e724d
Step 7/8 : RUN chmod +x /rbdplugin
 ---> Using cache
 ---> 276aa5b8880b
Step 8/8 : ENTRYPOINT ["/rbdplugin"]
 ---> Using cache
 ---> 6abe60c859e8
Successfully built 6abe60c859e8
Successfully tagged quay.io/cephcsi/rbdplugin:v0.3.0
docker push quay.io/cephcsi/rbdplugin:v0.3.0
The push refers to repository [quay.io/cephcsi/rbdplugin]
```

Reply:
Ouch! across branches this should be exporting ENV_* rather than just the *

Will update this PR and correct the same in other branches as well.

Reply:
Updated.